bazel/repositories.bzl

```diff
@@ -317,6 +318,13 @@ def _com_google_absl():
         actual = "@com_google_absl//absl/numeric:int128",
     )
 
+def _com_googlesource_boringssl():
+    _repository_impl("com_googlesource_boringssl")
```

Comment:
This is wrong. While you're trying to come up "semi-standard" name, you're ignoring the `boringssl` name provided in BoringSSL's [`WORKSPACE`](https://boringssl.googlesource.com/boringssl/+/master-with-bazel/WORKSPACE), which leads to a build time warning:

```
WARNING: /build/tmp/_bazel_bazel/436badd4919a15958fa3800a4e21074a/external/com_googlesource_boringssl/WORKSPACE:1: Workspace name in /build/tmp/_bazel_bazel/436badd4919a15958fa3800a4e21074a/external/com_googlesource_boringssl/WORKSPACE (@boringssl) does not match the name given in the repository's definition (@com_googlesource_boringssl); this will cause a build error in future versions
```

Reply:
Oops, I'd been looking at the chromium-stable branch and forgot WORKSPACE was populated in chromium-stable-with-bazel. Fixed.